peac/scripts/store_colors.py

```diff
@@ -0,0 +1,16 @@
+import sys
+import os
+import numpy as np
+from pypcd import pypcd
+
+if __name__ == '__main__':
+    pc = pypcd.PointCloud.from_path(sys.argv[1])
+    colors =  pc.pc_data["rgb"]
+    colors_map = {}
+    for n, color in enumerate(np.unique(colors)):
```

Comment:
I think it can be simplified:
```
    for n, color in enumerate(np.unique(colors)):
        colors[np.where(colors == color)] = n + 1
```

Reply:
Also looks like you don't do anything with unsegmented areas --- I think they have to be painted with the same color every time, and be detected here

Reply:
Also copy of array isn't necessary as you don't use array any more after changes

Reply:
If you want a new instance just for pretty code and another dtype of items, then better to use np.zeros_like()